config/default/kustomization.yaml

```diff
@@ -28,7 +28,7 @@ patchesStrategicMerge:
 images:
 - name: kusk-gateway
   newName: kubeshop/kusk-gateway
-  newTag: v1.2.5
+  newTag: v1.3.1
```

Comment:
Do we have v1.3.1?

Reply:
Not at the moment. We are currently  at `v1.3.0`.

What should I do about it?

Reply:
```suggestion
  newTag: v1.3.0
```

Reply:
Resolved.